Create a checklist for MDN short surveys (#17)
web-platform-dx/developer-research: short-survey-checklist.md

# Checklist for MDN short surveys

> This document lists the standard operating procedure for creating MDN short surveys. 

Reviewer: ```suggestion
This document lists points that need to be checked before publishing an MDN short survey. 
```

> # Mandatory fields

Reviewer: I realize these comes from the MDN short survey process, but suggestions below to improve a bit from there

> # Mandatory fields

Reviewer: ```suggestion
# Filing a survey request
```

- [ ] Topic: The topic of the survey

> - [ ] Category: E.g. CSS, JavaScript, Accessibility, documentation

Reviewer: is it the category of the pages on which the survey will be shown? if so, that should be clarified

> - [ ] Category: E.g. CSS, JavaScript, Accessibility, documentation

Reviewer: ```suggestion
- [ ] Areas of MDN where the survey should show: E.g. CSS, JavaScript, Accessibility
```

> - [ ] Date: When the survey should ideally be run

Reviewer: ```suggestion
- [ ] Date: Time constraints on when the survey should be run (if any)
```

- [ ] Objective: What is the goal?
- [ ] Justification: How is it relevant/interesting to the MDN audience?
- [ ] Actions: Actions which will be taken based on the results

> - [ ] Audience percentage: Suggested percentage of the MDN audience to be exposed to the survey. Default is 5%, if higher a clear motivation is needed.

Reviewer: ```suggestion
- [ ] Sampling rate: Suggested percentage of the MDN audience to be exposed to the survey. Default is 5%, if higher a clear motivation is needed.
```

- [ ] Questions: Suggested questions and answers

> - [ ] Other: Any other information which is relevant.

Reviewer: Probably not "mandatory" :)

> # Frequently requested information

Reviewer: ```suggestion
# Checking questions
```

- [ ] How should the questions be presented? Multiple choice, expanded list, etc
- [ ] Does any of the lists need to appear in random order?
- [ ] Do any of the items in a list need to appear in a specific order?
- [ ] Is there a free form field that needs to be excluded from random order?

> - [ ] Is there a gating question that should diqualify people from taking the survey?

Reviewer: ```suggestion
- [ ] Is there a gating question that should disqualify people from taking the survey?
```

> - [ ] Is there a gating question that should diqualify people from taking the survey?

Reviewer: The MDN short survey guidelines also specifically asked not to ask PII beyond what's included in the privacy policy for the survey takers
